docs/how-to-docs/keycloak-sidecar.md

```diff
@@ -0,0 +1,109 @@
+# Using Keycloak Sidecar
+
+The keycloak gatekeeper (also know as the keycloak sidecar) is used to enable SSO on applications. [More information on configuration options](https://www.keycloak.org/docs/8.0/securing_apps/#configuration-options)
+
+## Keycloak sidecar using single role
+
+Below is a example of a keycloak sidecar which uses a single role for all URIs.In this example the configuration options are passed in through environment variables and command arguments.
+
+```yaml
+- name: keycloak-gatekeeper
+  image: quay.io/keycloak/keycloak-gatekeeper:latest
```

Comment:
please use a named tag rather than latest - latest could result in sidecar proxy being ahead of the server version and cause issues. using named tags in deployments is better practice in any case.

Reply:
changed to 10.0.0 tag